src/current/v25.1/set-up-logical-data-replication.md

```diff
@@ -36,12 +42,6 @@ You'll need:
     - All nodes in each cluster will need access to the Certificate Authority for the other cluster. Refer to [Step 2. Connect from the destination to the source](#step-2-connect-from-the-destination-to-the-source).
 - LDR replicates at the table level, which means clusters can contain other tables that are not part of the LDR job. If both clusters are empty, create the tables that you need to replicate with **identical** schema definitions (excluding indexes) on both clusters. If one cluster already has an existing table that you'll replicate, ensure the other cluster's table definition matches. For more details on the supported schemas, refer to [Schema Validation](#schema-validation).
```

Comment:
@msbutler @alicia-l2 Am I right in thinking that the main point of this prerequisite bullet applies only to `CREATE LOGICAL REPLICATION STREAM`?

Reply:
> create the tables that you need to replicate with **identical** schema definitions (excluding indexes) on both clusters.

correct, this only applies to the original syntax.